Dockerfile

```diff
@@ -19,6 +19,20 @@ RUN yum install -y nodejs npm java-11
 # Copy the go source
 COPY ["Makefile", "main.go", ".openapi-generator-ignore", "openapitools.json", "./"]
 
+
+# Download protoc compiler v24.3
+RUN set -ex\
+    ; ARCH=$(uname -m) ; echo $ARCH \
+    ; if [ "$ARCH" == "s390x" ] ; then ARCH="s390_64" ; elif [ "$ARCH" == "ppc64le" ] ; then ARCH="ppcle_64" ; fi \
+    ; wget -q https://github.com/protocolbuffers/protobuf/releases/download/v24.3/protoc-24.3-linux-$ARCH.zip -O protoc.zip \
+    ; unzip -q protoc.zip \
+    ; bin/protoc --version \
+    ; rm protoc.zip \
+    ;
+
+# Download tools
+RUN make deps
```

Comment:
since this `RUN make deps` happens-before copying `COPY scripts/ scripts/` (line 27/41) it will make makefile fail to execute:

https://github.com/kubeflow/model-registry/blob/5f7fac0f1d98dc3daa0027801dc365ac7e26814c/Makefile#L115-L116

and it's actually a duplicate of line 33/47

Reply:
I will update install_protoc with other architecture.

Reply:
There is small confusion from my end. I have put my changes on 1 week old code . I will rebase my change on to the latest code & raise again. As part of that your abnove comments will be addressed

Reply:
Updated protoc.sh scripts for all architecture. @tarilabs  Please review.